src/main/java/org/umssdiplo/automationv01/core/utils/CommonEvents.java

```diff
@@ -154,14 +154,25 @@ public static void waitUntil(long waitTime) {
         }
     }
 
+    /***
+     * Verify if an atributte is present on web element
+     * @param webElement
+     * @param attr
+     * @return
+     */
     public static boolean isAttributPresent(WebElement webElement, String attr) {
         String value = webElement.getAttribute(attr);
         return value != null ? value.equals("true") : false;
     }
 
+    /***
+     * Verify if an element is displayed first to verify if is present
+     * @param webElement
+     * @return
+     */
     public static boolean elementIsPresent(WebElement webElement) {
         try {
-            return webElement.isDisplayed() ? CommonEvents.isPresent(webElement) : false;
+            return webElement.isDisplayed() ? isPresent(webElement) : false;
```

Comment:
we are keeping this implementation. please avoid to use this type of logic, because it is just for this specific escenario.